di-ipv-credential-issuer-stub/deploy/bav/template.yaml

```diff
@@ -605,6 +638,19 @@ Resources:
                   - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/stubs/credential-issuer/*"
                   - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/stubs/credential-issuer-stub-clients"
                   - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/stubs/credential-issuer-stub-clients/*"
+        - PolicyName: GetDynatraceSecret #pragma: allowlist secret
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action: secretsmanager:GetSecretValue #pragma: allowlist secret
+                Resource: arn:aws:secretsmanager:eu-west-2:216552277552:secret:* #pragma: allowlist secret
```

Comment:
This is only stubs so probably not a big deal, but should we restrict the secret path a bit here? (Similar to line 626 above)

Reply:
Also around line 589

Reply:
Im not sure the answer to these questions, this was just duplicated from the other CRI templates which work fine. All i've done is copy paste and rename the CRI its referencing

Reply:
@blakeyp are we OK to just copy these over, or should we spend some time potentially improving them?